src/Assert.php

```diff
@@ -817,6 +817,23 @@ public static function uuid($value, $message = '')
         }
     }
 
+    public static function throws(callable $expression, $expected, $message = '')
+    {
+        self::string($expected);
+        try {
+            $expression();
+        }
+        catch (\Exception $e) {
+            if (is_a($e, $expected, true)) {
```

Comment:
it's a reminiscence of my attempt to support both strings and instances. I can change it back maybe if it's clearer.